js/ecc/util.test.js

```diff
@@ -9,6 +9,33 @@ import libecc_module from "./libecc.js";
 import * as util from "./util.js";
 import assert from "assert";
 
+describe("util",() => {
+
+    it("ecc_memzero_test", async () => {
+        const libecc = await libecc_module();
+        let buf = new Uint8Array(100);
+        libecc.ecc_randombytes(buf, buf.length);
+        libecc.ecc_memzero(buf, buf.length);
+        let count = 0;
+        for (let i = 0; i < buf.length; i++) {
+            if (buf[i] === 0) count++;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/aldenml/ecc/pullRequest?prid=7807634)
```suggestion
            if (buf[i] === 0) {count++;}
```